docs/reference/query-dsl/vector-queries.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[vector-queries]]
+== Vector queries
+
+Vector queries search efficiently for vectors in order to perform <<semantic-search,semantic search>>.
```

Comment:
May be better phrase like this: "Vector queries are specialized queries that work on vector fields to efficiently perform semantic search"?

Reply:
I like that, thanks!